src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java

```diff
@@ -353,25 +348,23 @@ private JsonNode parseClientConfigureResponse(Map<String, Object> response) {
     return responseNode;
   }
 
-  private Map<String, Object> makeClientConfigureCall(Map<Object, Object> payload)
+  private ConfigureResponse makeClientConfigureCall(Map<Object, Object> payload)
       throws IOException {
     try {
 
-      Map<String, Object> response =
+      ConfigureResponse response =
           executeWithRetries(
-              Map.class,
+              ConfigureResponse.class,
               CLIENT_CONFIGURE_ENDPOINT,
               mapper.writeValueAsString(payload),
               "client configure",
               STREAMING_CLIENT_CONFIGURE,
               httpClient,
-              requestBuilder,
-              statusGetter);
```

Comment:
can we remove the statusGetter now or it's still used somewhere else?

Reply:
Sure, removed.